filebeat/crawler/prospector.go

```diff
@@ -5,14 +5,17 @@ import (
 	"sync"
 	"time"
 
+	"regexp"
+
 	cfg "github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester"
 	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
 type Prospector struct {
-	ProspectorConfig cfg.ProspectorConfig
+	ProspectorConfig prospectorConfig
```

Comment:
maybe name field `config`. having `prospector.ProspectorConfig` is pretty stuttery.